OneBlink.SDK.Tests/FormsAppsClientTests.cs

```diff
@@ -46,22 +46,13 @@ public void throws_error_if_keys_empty()
             Assert.Throws<ArgumentException>(() => new FormsAppsClient("", ""));
         }
 
-        [Fact]
```

Comment:
Why has this test been removed? Will these tests pass after the migration?

Reply:
Code no longer throws this error and yes.